### PR TITLE
Add UPGRADING.md

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level-hyper/issues/26

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Upgrade Guide
+
+This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
+
+## v2
+
+Dropped support for node 4. No other breaking changes.
```
Comment (ralphtheninja) [inline UPGRADING.md]: I should mention more node versions here.

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Upgrade Guide
+
+This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
+
+## v2
+
+Dropped support for node 0.12, 4, 5 and 7.
+
+This is an upgrade to `leveldown-hyper@~2.0.0` which is based on `abstract-leveldown@~5.0.0` which in turn contains breaking changes to [`.batch()`](https://github.com/Level/abstract-leveldown/commit/a2621ad70571f6ade9d2be42632ece042e068805). Though this is negated by `levelup`, we decided to release a new major version in the event of dependents reaching down into `db.db`.
```
Comment (vweevers) [inline UPGRADING.md]: Maybe: which is based on => which depends on

Rest LGTM

File: UPGRADING.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+The parts that make up `level` have been refactored to increase modularity. This is an upgrade to `level-packager@~4.0.0`, which in turn upgraded to `levelup@^3.0.0`. The responsibility of encoding keys and values moved from [`levelup`](https://github.com/Level/levelup) to [`encoding-down`](https://github.com/Level/encoding-down), which comes bundled with [`level-packager`](https://github.com/Level/packager).
```
Comment (vweevers) [inline UPGRADING.md]: `level` => `level-hyper`. Same below

Comment (ralphtheninja) [reply UPGRADING.md]: What would I do without you @vweevers ? :)

File: UPGRADING.md [hunk 4]
```diff
@@ -6,11 +6,11 @@ This document describes breaking changes and how to upgrade. For a complete list
...
+The parts that make up `level-hyper` have been refactored to increase modularity. This is an upgrade to `level-packager@~4.0.0`, which in turn upgraded to `levelup@^3.0.0`. The responsibility of encoding keys and values moved from [`levelup`](https://github.com/Level/levelup) to [`encoding-down`](https://github.com/Level/encoding-down), which comes bundled with [`level-packager`](https://github.com/Level/packager).
 
-Being a convenience package, `level` glues the parts back together to form a drop-in replacement for the users of `levelup@1`, while staying fully compatible with `level@1`. One thing we do get for free, is native Promise support.
+Being a convenience package, `level-hyper` glues the parts back together to form a drop-in replacement for the users of `levelup@1`, while staying fully compatible with `level@1`. One thing we do get for free, is native Promise support.
```
Comment (ralphtheninja) [inline UPGRADING.md]: Hmm, this should probably be `levelup` no?

**Pull request merged**